browser/extensions/brave_extensions_browser_client_impl.cc

```diff
@@ -7,15 +7,23 @@
 
 #include <memory>
 
+#include "base/macros.h"
 #include "brave/browser/extensions/brave_extensions_browser_api_provider.h"
 #include "brave/browser/profiles/profile_util.h"
+#include "chrome/browser/extensions/chrome_component_extension_resource_manager.h"
 #include "components/prefs/pref_service.h"
 
 namespace extensions {
 
 BraveExtensionsBrowserClientImpl::BraveExtensionsBrowserClientImpl() {
   BraveExtensionsBrowserClient::Set(this);
   AddAPIProvider(std::make_unique<BraveExtensionsBrowserAPIProvider>());
+  // ChromeComponentExtensionResourceManager's Data needs to be LazyInit'ed on
+  // the UI thread (due to pdf_extension_util::AddStrings calling
+  // g_browser_process->GetApplicationLocale() that has a DCHECK to that
+  // regard).
+  ignore_result(GetComponentExtensionResourceManager()
```

Comment:
I'm confused, why is this only a problem for brave and not chrome?

Reply:
it seems like this defeats the purpose of the lazy init

Reply:
In chrome it gets called by `CryptoTokenExtension` extension initialization early on from the UI thread. But we don't load that extension.

Reply:
Follow up https://github.com/brave/brave-browser/issues/11120

Reply:
it's interesting because it seems like both for us (with this fix) and in Chrome, the lazy init is basically negated by something calling it during startup